integrations/tensorflow/compiler/BUILD

```diff
@@ -119,6 +119,30 @@ cc_binary(
     }),
 )
 
+cc_binary(
+    name = "iree-import-tflite",
+    srcs = ["iree-import-tflite-main.cpp"],
+    deps = [
+        "@llvm-project//llvm:Support",
+        "@llvm-project//mlir:IR",
+        "@llvm-project//mlir:Support",
+        "@llvm-project//mlir:QuantOps",
+        "@llvm-project//mlir:StandardOps",
+        "@llvm-project//mlir:TosaDialect",
+        "@org_tensorflow//tensorflow/compiler/mlir/lite:flatbuffer_import",
+        "@org_tensorflow//tensorflow/compiler/mlir/lite:tensorflow_lite",
+        "@org_tensorflow//tensorflow/compiler/mlir/tosa:tosa_legalize_tfl",
```

Comment:
Nice to see TOSA as part of IREE ! 

Would it not be sufficient to add tfl_tosa_passes as a single dependency here ? This is the pass pipeline that includes multiple TFL->TOSA mapping passes that are run in sequence. It subsumes the tosa_legalize_tfl dependency.

Reply:
Yeah, we're excited :)

It should be sufficient, but we got some of the BUILD dep layering wrong in the TF repo, and since tf-opt is a mondo library that links everything in, it worked. I added the ones below to overcome link errors and need to send a change upstream to do it right.

Reply:
There are also some other misc build fixes I noticed along the way, so I'll take care of it. Mainly: we can make some of the deps more fine-grained so that for TFLite, we save about ~30MB of binary size (presently >50MB, down to ~20MB or so)

Reply:
Sounds great! We're happy to take any advice on avoiding the use tf-opt, while still getting the TF protobuf to MLIR serialization pieces we need to make the TOSA path work. Right now from protobuf to TOSA MLIR form, the path is

$ tf-mlir-translate --graphdef-to-mlir --tf-enable-shape-inference-on-import {additional input/output params} some.pb | tf-opt --tf-executor-to-functional-conversion | tf-opt --tf-to-tosa-pipeline  

Since the earliest days of work on TOSA, we have been interested in keeping the binary size down somehow, too. However the above command sequence compels us to use the heavyweight tf-opt anyway.

Reply:
I can do some surgery on that to get it chopped down for you. The TF side will remain heavy (but not as heavy as it is now) due to constant folding needing so many CPU kernels, but I've got some recipes that have been effective on our side for managing that (i.e. I'm down to about a 4-5m clean rebuild time of just what is needed for these frontend bits).

Reply:
Thank you! That would be a huge gain for us , and would also benefit you if you’re trying to process TF protobufs through TOSA using this same path.